pkg/client/tx/client.go

```diff
@@ -135,33 +135,33 @@ func NewTxClient(
 	deps depinject.Config,
 	opts ...client.TxClientOption,
 ) (_ client.TxClient, err error) {
-	tClient := &txClient{
+	txClient := &txClient{
```

Comment:
`txClient := &txClient{`

I wanted to check whether this causes compilation issues or not?

Reply:
Yes it compiles, golang differenciates the variable from the type.